assets/src/dashboard/components/cardGridItem/test/cartTitle.js

```diff
@@ -50,4 +51,19 @@ describe('CardTitle', () => {
     expect(getByTestId('title-rename-input')).toBeDefined();
     expect(getByTestId('title-rename-input')).toHaveFocus();
   });
+
+  it(`should prepend "draft" before displayDate when status is ${STORY_STATUS.DRAFT}`, () => {
+    const { getByText } = renderWithTheme(
+      <CardTitle
+        title="Sample Story"
+        displayDate="Modified July 13"
+        status={STORY_STATUS.DRAFT}
+        onEditCancel={jest.fn()}
```

Comment:
Looks like a missed comment, but this should drop the `()`. It has no functional difference since this would be effectively passing undefined, but we don't want to call the function inline.

Reply:
whoops! yup. I'll fix this file, thanks for noting this.